content/docs/tasks/policy-enforcement/rate-limiting/index.md

```diff
@@ -55,13 +55,113 @@ so the configuration to enable rate limiting on both adapters is the same.
         * `memquota adapter` defines memquota adapter configuration.
         * `quota rule` defines when quota instance is dispatched to the memquota adapter.
 
-    Run the following command to enable rate limits.
+    Run the following command to enable rate limits using memquota.
 
     {{< text bash >}}
     $ kubectl apply -f @samples/bookinfo/policy/mixer-rule-productpage-ratelimit.yaml@
     {{< /text >}}
 
-1. Confirm the `memquota` handler was created:
+    Or
+
+    Save the following yaml file as `redisquota.yaml`. Replace [rate_limit_algorithm](/docs/reference/config/policy-and-telemetry/adapters/redisquota/#Params-QuotaAlgorithm),
+    [redis_server_url](/docs/reference/config/policy-and-telemetry/adapters/redisquota/#Params) with values for your configuration.
+
+    {{< text yaml >}}
```

Comment:
Does this yaml content exist on GitHub? If so, we should just link to that instead.

Reply:
No, it doesn't... and guess it's too late to add to istio 1.0 now..